### DISCUSSION TITLE
Invoice "update" coverage

### DISCUSSION THREAD

**Discussion by glaidler** [codatio/codat-docs#805] [Q&A]
Hi,

We're currently evaluating codat.  According to this data coverage explorer, it suggests that only MYOB, Quickbooks and Xero can have their Invoice records updated to "Paid" after they have been "Created".

https://knowledge.codat.io/supported-features/accounting?view=tab-by-data-type&dataType=invoices

Am I understanding this correctly, or does the "No update" still allow you to allocate payment allocations to it?

**Comment by dcoplowe** [comment]
Hi @glaidler,

We currently support updating invoices in MYOB, Quickbooks, Xero, Sage 50 (UK) and Sage Business Cloud Accounting where the last two are in a beta release whilst we monitor them for bugs.

Payment allocations against invoices can be against an invoice using the [payments endpoint](https://knowledge.codat.io/supported-features/accounting?view=tab-by-data-type&dataType=invoices) once you have created the invoice.

Let me know if you'd like to talk to a specialist about this. 

Many thanks,

David
